### DISCUSSION TITLE
Where to get arch logo SVG for neofetch?

### DISCUSSION THREAD

**Discussion by zrin9021** [prasanthrangan/hyprdots#287] [Q&A] [answered]
Hi, where can I get the arch logo image for neofetch like in this picture? 
![image](https://github.com/prasanthrangan/hyprdots/assets/85588081/9ef85bb8-a740-4321-9718-f177061eacd5)

**Comment by T-Crypt** [comment, marked as answer]
Here you go.

![arch](https://github.com/prasanthrangan/hyprdots/assets/132922589/8cdab80f-e094-4eef-b9fb-beb328122b1f)

**Reply by prasanthrangan** [reply]
Hi @zrin9021 , are you looking for the svg file for this or just the png file?

**Reply by zrin9021** [reply]
> Hi @zrin9021 , are you looking for the svg file for this or just the png file?

Hi. Just the png file. I updated hyprdots and it looks like it was added to the neofetch pngs. Thank you.
